### PR TITLE
Increase end-to-end test coverage

### PR DESCRIPTION
**Pull request by zeeke**
Add a basic assertion in the end-to-end suite to
verify that the config-daemon can publish
k8s Events.
cc @rollandf 

Add an end-to-end test to verify that configuring the LogLevel
field of a SriovNetwork makes sriov cni Add command logging
to multus pods
cc @andreaskaris

### CODE REVIEW

**Review by andreaskaris — CHANGES_REQUESTED** [review]
Looking good for the LogLevel stuff minus the "FIt" / "It" in test_sriov_operator.go on line 111

File: test/conformance/tests/test_sriov_operator.go [hunk 1]
```diff
@@ -108,7 +108,7 @@ var _ = Describe("[sriov] operator", func() {
 	Describe("No SriovNetworkNodePolicy", func() {
 		Context("SR-IOV network config daemon can be set by nodeselector", func() {
 			// 26186
-			FIt("Should schedule the config daemon on selected nodes", func() {
+			It("Should schedule the config daemon on selected nodes", func() {
```
[the remark this answers is not in the document]
Comment (zeeke) [reply test/conformance/tests/test_sriov_operator.go:111]: sure, it's better to make it clean

**Comment by andreaskaris** [comment]
From my side (sriov logging) lgtm

File: test/conformance/tests/test_sriov_operator.go [hunk 2]
```diff
@@ -913,6 +930,36 @@ var _ = Describe("[sriov] operator", func() {
...
+
+				testPod := createTestPod(node, []string{sriovNetworkName})
+
+				recentMultusLogs := getMultusPodLogs(testPod.Spec.NodeName, podDeployTime)
+
+				Expect(recentMultusLogs).To(
+					ContainElement(
+						// Assert against multiple ContainSubstring condition because we can't make assumption on the order of the chunks
+						And(
+							ContainSubstring(`level="debug"`),
```
Comment (mlguerrero12) [inline test/conformance/tests/test_sriov_operator.go:954]: the problem with this is that a simple change of the log message in the sriov-cni will break ci here. Perhaps we could use something that is less likely to change. The first message of cmdAdd for example. What do you think?

	logging.Debug("function called",
		"func", "cmdAdd",
		"args.Path", args.Path, "args.StdinData", string(args.StdinData), "args.Args", args.Args)

Comment (zeeke) [reply test/conformance/tests/test_sriov_operator.go:954]: sounds good

**Review by adrianchiris — APPROVED** [review]

**Comment by mlguerrero12** [comment]
LGTM

**Review by SchSeba — APPROVED** [review]

**Pull request merged**